### PR TITLE
doc(telemetry): collect SPDK node disk type

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/6033

### CODE REVIEW

File: content/docs/1.6.0/references/settings.md [hunk 1]
```diff
@@ -253,6 +254,7 @@ This information will help us gain insights how Longhorn is being used, which wi
 - Longhorn namespace UID.
 - Number of Longhorn nodes.
 - Number of volumes of each access mode (RWO, RWX, unknown).
+- Number of volumes of each backup storage driver (v1, v2).
```
Comment (innobead) [inline content/docs/1.6.0/references/settings.md:257]:
```suggestion
- Number of volumes of the backend storage driver in each data engine (v1, v2).
```

**Review by innobead — CHANGES_REQUESTED** [review]

File: content/docs/1.6.0/references/settings.md [hunk 2]
```diff
@@ -252,6 +252,7 @@ This information will help us gain insights how Longhorn is being used, which wi
 **Node Information collected from all cluster nodes includes:**
 - Number of disks of each device type (HDD, SSD, NVMe, unknown).
   > This value may not be accurate for virtual machines.
+- Number of disk of each Longhorn disk type (block, filesystem).
```
Comment (jillian-maroket) [inline content/docs/1.6.0/references/settings.md:255]:
```suggestion
- Number of disks for each Longhorn disk type (block, filesystem).
```

File: content/docs/1.6.0/references/settings.md [hunk 3]
```diff
@@ -260,6 +261,7 @@ This information will help us gain insights how Longhorn is being used, which wi
 - Longhorn namespace UID.
 - Number of Longhorn nodes.
 - Number of volumes of each access mode (RWO, RWX, unknown).
+- Number of volumes of the backend storage driver in each data engine (v1, v2).
```
Comment (jillian-maroket) [inline content/docs/1.6.0/references/settings.md:264]:
```suggestion
- Number of volumes associated with the backend store driver in each data engine (v1, v2).
```

**Review by jillian-maroket — COMMENTED** [review]
Just a few nits.

**Review by jillian-maroket — APPROVED** [review]

**Review by innobead — APPROVED** [review]

**Pull request merged**